backup/setup/systems.json

```diff
@@ -73,13 +89,94 @@
 			"$date": "2022-07-12T14:01:58.588Z"
 		},
 		"type": "oidc",
-		"alias": "iserv",
+		"alias": "oidcmock",
 		"config": {
-			"clientId": "ISERV_CLIENT_ID",
-			"clientSecret": "ISERV_CLIENT_SECRET",
-			"authorizationUrl": "https://iserv.n21.dbildungscloud.de/iserv/auth/auth",
-			"tokenUrl": "https://iserv.n21.dbildungscloud.de/iserv/auth/public/token",
-			"logoutUrl": "https://iserv.n21.dbildungscloud.de/iserv/auth/logout"
+			"clientId": "${ OIDCMOCK__CLIENT_ID }",
+			"clientSecret": "${ OIDCMOCK__CLIENT_SECRET }",
+			"authorizationUrl": "${ OIDCMOCK__BASE_URL }/connect/authorize",
+			"tokenUrl": "${ OIDCMOCK__BASE_URL }/connect/token",
+			"logoutUrl": "${ OIDCMOCK__BASE_URL }/connect/endsession",
+			"userinfoUrl": "${ OIDCMOCK__BASE_URL }/connect/userinfo",
+			"defaultScopes": "openid profile email"
+		}
+	},
+	{
+		"type" : "ldap",
+		"alias" : "schoolOne0",
+		"ldapConfig" : {
+			"active" : true,
+			"federalState": {
+			"$oid": "0000b186816abba584714c56"
+			},
+			"url" : "ldap://sc-openldap-single-svc.sc-common.svc.cluster.local:389",
+			"rootPath" : "o=schoolOne0,dc=de,dc=example,dc=org",
+			"searchUser" : "cn=admin,dc=example,dc=org",
+			"searchUserPassword" : "${SC_COMMON_LDAP_PASSWORD}",
```

Comment:
This will be overwritten by the next export as well.

Reply:
same as above